__tests__/mocks/hospitals.ts

```diff
@@ -70,3 +72,67 @@ export const hospitalsResponse = {
     },
   ],
 };
+
```

Comment:
isn't it possible to use the same hospital data?  
`
import hospitalResponse from "../mocks/hospitals"
const hospital = hospitalResponse.value[0]
render(<RecomendedHospital institution={hospital} />)
`
or test